book/src/install.md

```diff
@@ -54,9 +54,3 @@ This will install the `hx` binary to `$HOME/.cargo/bin`.
 Helix also needs it's runtime files so make sure to copy/symlink the `runtime/` directory into the
 config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
 via the `HELIX_RUNTIME` environment variable.
-
-## Building tree-sitter grammars
-
-Tree-sitter grammars must be fetched and compiled if not pre-packaged.
-Fetch grammars with `hx --grammar fetch` (requires `git`) and compile them
-with `hx --grammar build` (requires a C compiler).
```

Comment:
It's good to have this documented, because it shows how to compile user-specified grammars. The current build script runs these commands for you for backwards compatibility reasons but I don't want to rely on that.